src/lib/y2storage/boot_requirements_strategies/uefi.rb

```diff
@@ -39,10 +39,16 @@ def needed_partitions(target)
       # @return [Array<SetupError>]
       def warnings
         res = super
+        textdomain "storage"
```

Comment:
Isn't this a weird location to call textdomain?

Reply:
well, as it is only place calling localized message it is fine for me, but I can create initialize there if needed.

Reply:
This will break as soon as you add another message, and forget to update the textdomain since a grep will show it is already there... please move it.